airflow/executors/kubernetes_executor.py

```diff
@@ -372,7 +374,8 @@ def __init__(self,
         self.log.debug("Kubernetes using namespace %s", self.namespace)
         self.kube_client = kube_client
         self.launcher = PodLauncher(kube_client=self.kube_client)
-        self.worker_configuration = WorkerConfiguration(kube_config=self.kube_config)
+        # Parse the airflow.cfg once on AirflowKubernetesScheduler initialization
+        self.worker_configuration = WorkerConfiguration(kube_config=self.kube_config).make_pod()
```

Comment:
This is variable is now somewhat confusingly named -- it's no longer a WorkerConfiguration but a v1.Pod object.